override-THIRD-PARTY.properties

```diff
@@ -214,6 +216,8 @@ org.latencyutils--LatencyUtils--2.0.3=BSD-2-Clause
 org.openpnp--opencv--3.2.0-1=BSD-3-Clause
 # https://repo1.maven.org/maven2/org/osgi/org.osgi.core/4.2.0/org.osgi.core-4.2.0.jar
 org.osgi--org.osgi.core--4.2.0=Apache-2.0
+# https://gitlab.ow2.org/asm/asm/-/blob/ASM_5_0_4/LICENSE.txt
+org.ow2.asm--asm--5.0.4
```

Comment:
This is missing the license name. The tests passed - oops.

Reply:
Fixed